src/components/style/Flashlight.css

```diff
@@ -1,26 +1,17 @@
 
 #container {
-    margin-left: 100px; /*this is just for an offset*/
+    
     position: relative;
     overflow: hidden;
     filter: brightness(0%);
-    /* max-width: min-content;
-    max-height: min-content;
-    display: inline-block; */
-    background-image: url('../../koala.jpg');
+    background-image: url('../../img/koala.jpg');
```

Comment:
Didn't catch this earlier, but this is slightly bad practice - hard coding the `../../` may cause issues if we switch up the domain (e.g. if we do `nofilter.uclaacm.com`). I would either [import the image into your code](https://create-react-app.dev/docs/adding-images-fonts-and-files/) (and set the `background-image` with CSS-in-JS) or use [the public folder](https://create-react-app.dev/docs/using-the-public-folder).